Assets/Scripts/Gamemode/Racing/RacingObjective.cs

```diff
@@ -78,6 +79,7 @@ private void Start()
             }
             Gamemode.Objectives.Add(this);
             State = ObjectiveState.Queue;
+            audioSource = GetComponent<AudioSource>();
```

Comment:
There is no component "AudioSource" on the RacingObjective prefab. Please add this Component to the prefab, otherwise line 119 `audioSource.Play()` will throw a NullReferenceException. Instead of using GetComponent, you can also directly assign this inside Unity by using: `[SerializeField] private AudioSource audioSource;`